### PR TITLE
KK 004 -  Home Page - Changed logo and welcome msg🐨

### PR DESCRIPTION
**Pull request by eleniaslanidou89**
Home Page - Changed logo and welcome msg🐨
image

### CODE REVIEW

File: src/components/HomePage.vue [hunk 1]
```diff
@@ -50,7 +50,8 @@ export default {
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
 .logo {
-  border-radius: 50%;
+  width: 250px;
```
Comment (kgkoutio) [inline src/components/HomePage.vue:53]: Is it a common practice in web development to set images with fixed width and fixed height? How does that scale in the mobile browsers where there might not be enough space?

**Review by kgkoutio — COMMENTED** [review]
Hey @eleniaslanidou89 ! Great work with changing the logo 💯  I have a few thoughts when it comes to the code review based on my experience:
Branch naming is good to contain the ticket id but also a descriptive message, as it will be also humanly readable. This will help the reviewer to check out the right branch, and avoid mistakes. In that case I would name the branch `KK-004-new-logo`. What do you think? (Of course you don't have to change it now :) )

The other thought I have is added as a question.

**Review by kgkoutio — APPROVED** [review]

File: src/assets/giga-vue.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...ined-Shape" class="cls-2" d="M182,63c80.49,0,145.52,91.78,162.77,186h0l-30.24-14.59a7,7,0,0,0-7.2.67l-15,11.07-16-3.24v-72.7a7,7,0,0,0-7-7H94.67a7,7,0,0,0-7,7v72.7l-16,3.24-15-11.07a7,7,0,0,0-7.2-.67L19.24,248.94C36.5,154.76,101.51,63,182,63Z"/><path id="Combined-Shape-2" data-name="Combined-Shape" class="cls-3" d="M312.55,249l34.82,16.8a259.46,259.46,0,0,1,1.94,31.15C349.31,417.9,262.88,461,182,461S14.69,417.9,14.69,296.93a258.25,258.25,0,0,1,2-31.2L51.38,249h.56l13.9,10.25a7,7,0,0,0,5.55,1.22l16.28-3.29v54a7,7,0,0,0,7,7H269.33a7,7,0,0,0,7-7v-54l16.28,3.29a7,7,0,0,0,5.55-1.22L312,249Z"/><path id="Path-2" data-name="Path" d="M332.31,289.93a7,7,0,0,0-7,7A173,173,0,0,1,318,348.67a7,7,0,0,0,4.58,8.77,6.53,6.53,0,0,0,2.1.32,7,7,0,0,0,6.67-4.9,186.51,186.51,0,0,0,8-55.93,7,7,0,0,0-7-7Z"/><path id="Shape-2" data-name="Shape" d="M243.5,362.33a19,19,0,1,0,19,19,19,19,0,0,0-19-19Zm0,24a5,5,0,1,1,5-5A5,5,0,0,1,243.5,386.33Z"/><path id="Shape-3" data-name="Shape" d="M182,384a19,19,0,1,0,19,19,19,19,0,0,0-19-19Zm0,24a5,5,0,1,1,5-5,5,5,0,0,1-5,5Z"/><path id="Shape-4" data-name="Shape" d="M120.5,362.33a19,19,0,1,0,19,19A19,19,0,0,0,120.5,362.33Zm0,24a5,5,0,1,1,5-5,5,5,0,0,1-5,5Z"/></g></g></svg>
```
Comment (kgkoutio) [inline src/assets/giga-vue.svg:1]: Ah I forgot to ask about this one. Excuse my ignorance, but what is this for?

**Comment by kkarayannis** [comment]
When there are visual changes, a screenshot in the PR helps the reviewer a lot 🙂

File: src/components/HomePage.vue (hunk 1 above)
Comment (eleniaslanidou89) [reply src/components/HomePage.vue:53]: I agree with you @kgkoutio branch name should contain a descriptive message. 
According to the width and height in web development it is usually used to set a width and a height and then change it in the responsive part. I haven't add the responsive part yet. In the responsive part we can set the size of the image for a Tablet version ,mobile version etc. This is based my knowledge in React framework, I will check if there are differences in Vuejs.
Example: 
/***************   MOBILE VIEW   ***************/
.nav_logo {
    margin: 0 25px 0 10%;
    height: 60px;
  }

File: src/assets/giga-vue.svg (hunk 2 above)
Comment (eleniaslanidou89) [reply src/assets/giga-vue.svg:1]: The logo that we have is a combination of an img and a logo that I created. This svg is the img without the logo. I will delete it, we don't need it.

**Comment by eleniaslanidou89** [comment]
image

Changes on Home Page:
- Deleted the useless logo.
- Changed the welcome text and the styling of it.
- Changed the names of the classes (based on BEM).
- Improved the structure.

**Pull request merged**